### ISSUE TITLE
Consider using a different yum module

### ISSUE THREAD

**Issue by juniorsysadmin** [voxpupuli/puppet-php#284]
Perhaps the one in this namespace

**Comment by amette** [comment]
I would highly appreciate that as well. I have many servers that run PHP as well as elastic (former elasticsearch). I can't configure those machines via puppet, because the example42-yum dependency in this module clashes with the puppet-yum one in the elastic/elasticsearch puppet module.

**Comment by bluridium** [comment]
Agreed. The existing redhat.pp also hardcodes the remi56 repo so never allows you to install a different version.

I confirmed in a fork that yumrepo{} works fine and completely removes the dependence on example42-yum.

**Comment by danielsreichenbach** [comment]
Would highly agree with using <https://forge.puppet.com/puppet/yum> instead. example42-yum also introduces conflicts with many other modules expecting the yum module from voxpupuli.

**Comment by mbrodala** [comment]
Same here, I even had to fork this repository to drop this dependency. We are on Ubuntu thus don't need YUM anyways.

**Comment by kenyon** [comment]
Done in #355.
